cmd/incusd/api_cluster.go

```diff
@@ -3583,6 +3590,14 @@ func restoreClusterMember(d *Daemon, r *http.Request) response.Response {
 			}
 		}
 
+		logger.Infof("Initializing networks")
```

Comment:
And here the reverse is true, networks need to be brought back up prior to any instance being started back.

Reply:
Should the networks be started before or after the migration/instance reloading?

Reply:
Before